rpm/firejail.spec

```diff
@@ -48,6 +50,7 @@ Requires: %{name} = %{version}-%{release}
 %make_install
 rm -rf %{buildroot}%{_datadir}/bash-completion
 rm -rf %{buildroot}%{_datadir}/zsh/site-functions/_firejail
+install %{buildroot}/etc/firejail/firecfg.config %{buildroot}/usr/lib64/firejail/
```

Comment:
This looks a little suspicious. Why do we need to do this?

Reply:
I added the line because the build would shit itself with:
`error: File not found: /home/deploy/installroot/usr/lib64/firejail/firecfg.config`
until I did.

Reply:
It's already listed as excluded in the spec for firejail %files. Should it actually be excluded also from profiles?

Reply:
Not sure how relevant the file is to our usecase, but at least src/firecfg/main.c still contains code for parsing this file...

Reply:
> It's already listed as excluded in the spec for firejail %files. Should it actually be excluded also from profiles?

So what used to be packaged was default firecfg.config in profiles subpackage in case someone happened to need it for something. That subpackage is normally not included in installation and you don't really need it on Sailfish unless you are experimenting with firejail. If upstream moved it from `%_libdir` to `%_sysconfdir` then they did. I'm not sure if that's the right move, but even if we want it in `%_libdir` then, we should just `mv` it to the right place. (If there is no build option for example.)

Also you can't hardcode `lib64` like that. It would fail in 32-bit target.